lib/spoom/sorbet/config.rb

```diff
@@ -77,6 +88,9 @@ def parse_string(sorbet_config)
               if ignore
```

Comment:
If more than one of `ignore`, `extension`, and `skip` are true at the same time due to poor ordering or an accidental extra option in the config file then the "closing" of that flag then I think it will be missed and may mess something up later. Is there any initial verification of the structure of the config file when it's first processed?